db/service_updates.yml

```diff
@@ -0,0 +1,5 @@
+- id: 1
+  date: '2021-09-01'
```

Comment:
Is it possible to set a default of the date this gets published?

Reply:
@ralph-hawkins 
As we are using YAML file, we won't be able to set the date automatically.
We'll have to set it manually each time.

Reply:
can we set this elsewhere and only overwrite it if there is a value in the YAML?